src/azure-cli/azure/cli/command_modules/backup/custom_base.py

```diff
@@ -353,6 +353,8 @@ def resume_protection(cmd, client, resource_group_name, vault_name, container_na
     policy = show_policy(protection_policies_cf(cmd.cli_ctx), resource_group_name, vault_name, policy_name)
     custom_help.validate_policy(policy)
 
+    if item.properties.backup_management_type.lower() == "azureiaasvm":
+        return custom.resume_protection(cmd, client, resource_group_name, vault_name, item, policy)
```

Comment:
Use `elseif` for the condition branches below this one.

Reply:
This is a pylint violation. It doesn't allow having elif statements when we have if blocks returning something. It throws this error when i run a style check.
`R1705: Unnecessary "elif" after "return" (no-else-return)`

Reply:
Got it.